cypress/integration/v2/features/dynamic-content/current-post.spec.js

```diff
@@ -0,0 +1,326 @@
+
+/**
+ * External dependencies
+ */
+import { registerTests } from '~stackable-e2e/helpers'
+import { containsRegExp } from '~common/util'
+
+describe( 'Dynamic Content Current Post', registerTests( [
+	matchPostDataValues,
+	adjustFieldOptions,
+	adjustFieldValues,
+	assertEmptyValues,
+] ) )
+
+const fields = {
+	'Post Title': 'title',
+	'Post URL': 'link',
+	'Post ID': 'id',
+	'Post Slug': 'slug',
+	'Post Excerpt': 'excerpt',
+	'Post Date': 'date',
+	'Post Date GMT': 'date_gmt',
+	'Post Modified': 'modified',
+	'Post Modified GMT': 'modified_gmt',
+	'Post Type': 'type',
+	'Post Status': 'status',
+	'Author Name': 'name',
+	'Author ID': 'author',
+	'Author Posts URL': 'url',
+	'Comment Number': 'comments_num',
+	'Comment Status': 'comment_status',
+	'Featured Image URL': 'featured_image_urls',
+	'Author Profile Picture URL': '', // TODO: fields from here are not in getCurrentPostData
```

Comment:
what do you mean by this comment?

Reply:
the fields starting from `Author Profile Picture URL` is not being asserted here:

https://github.com/gambitph/Stackable-e2e-tests/pull/156/files#diff-ae6db55ead66eb837dac0a66d2a6d8d3cb2d8eccd993473426c3a82e7116e0ecR96-R114

this is because the contents of these fields are not readily available when we do a `cy.getPostData().then()...`

Reply:
since we need to complete all `TODOS` we should do this